jdaviz/configs/specviz2d/helper.py

```diff
@@ -149,14 +149,19 @@ def load_data(self, spectrum_2d=None, spectrum_1d=None, spectrum_1d_label=None,
                 # though _trace_dataset_selected was called internally to set them to reasonable
                 # new defaults.  We'll just call it again manually.
                 spext._trace_dataset_selected()
-                _ = spext.create_extract(add_data=True)
-
-                # Warn that this shouldn't be used for science
-                msg = ("The extracted 1D spectrum was generated automatically."
-                       " See the spectral extraction plugin for details or to"
-                       " perform a custom extraction.")
-                msg = SnackbarMessage(msg, color='warning', sender=self,
-                                      timeout=10000)
+                try:
+                    _ = spext.create_extract(add_data=True)
```

Comment:
Might as well clean it up a bit while you are at it?
```suggestion
                    spext.create_extract(add_data=True)
```